src/diffusers/models/prior_transformer.py

```diff
@@ -32,7 +33,7 @@ class PriorTransformerOutput(BaseOutput):
     predicted_image_embedding: torch.FloatTensor
 
 
-class PriorTransformer(ModelMixin, ConfigMixin):
+class PriorTransformer(ModelMixin, ConfigMixin, UNet2DConditionLoadersMixin):
```

Comment:
I'm a little bit surprised to see `UNet2DConditionLoadersMixin` works out of box for `PriorTransformer`:)

Reply:
Just `diffusers` things 😎